src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -297,6 +320,12 @@ function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
         $(`#responseCommentTable${id}`).css('margin-top', '0');
         $(`#responseCommentTable${id}`).hide();
     }
+    const checkboxesInAddFormTable = getCheckBoxesInFormTable($(`#showResponseCommentAddForm${id} > form`));
+    const valueOfCheckboxes = getInitialVisibilityOfCheckboxes($(`#showResponseCommentAddForm${id}`));
+    $.each(checkboxesInAddFormTable, (i, checkboxInAddFormTable) => {
+        const checkbox = `${$(checkboxInAddFormTable).attr('class')} - ${$(checkboxInAddFormTable).val()}`;
+        $(checkboxesInAddFormTable).prop('checked', valueOfCheckboxes.get(checkbox));
```

Comment:
Did you mean `$(checkboxInAddFormTable).prop(...)` instead of `$(checkboxesInAddFormTable).prop(...)`? If I understand your code correctly, it seems that you want to change the value of a single checkbox in this line.

This could be the source of the bug.